traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -174,7 +175,21 @@ func Read(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		resolvers = append(resolvers, resolver)
+		// Based on version we load types - for version 5 and above we use FederationResolverV5
+		if inf.Version.GreaterThanOrEqualTo(&api.Version{Major: 5, Minor: 0}) {
+			var v5Resolver tc.FederationResolverV5
+
+			// Convert FederationResolver fields to FederationResolverV5 fields
+			v5Resolver.ID = resolver.ID
```

Comment:
This looks good. Do you mind making it into a utility function please, so that future development can make use of it and not have to repeat these lines everywhere?